api/desecapi/models.py

```diff
@@ -378,8 +378,12 @@ def update_delegation(self, child_domain: Domain):
             raise ValueError('Cannot update delegation of %s as it is not an immediate child domain of %s.' %
                              (child_domain.name, self.name))
 
+        # Always remove delegation so that we con properly recreate it
```

Comment:
I'm wondering if this incurs significant overhead for dyn IP updates?

Reply:
No, that code only runs when creating or deleting a domain (not when doing an IP update)